### PR TITLE
[ADD] Add pos_analytic_config module.

### PR DESCRIPTION
**Pull request by adrienpeiffer**
Use analytic account defined on POS configuration for invoices and accounting entries generated form POS orders.

### CODE REVIEW

File: pos_analytic_config/README.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+POS Analytic Config
+===================
+
+This module was written to allow to use analytic account defined on POS configuration
```
Comment (pedrobaeza) [inline pos_analytic_config/README.rst]: Use the simpler form "This module allows to use"

File: pos_analytic_config/README.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+for invoices and accounting entries generated form POS orders.
```
Comment (pedrobaeza) [inline pos_analytic_config/README.rst]: s/form/from

File: pos_analytic_config/README.rst~ [hunk 3]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (legalsylvain) [inline pos_analytic_config/README.rst~]: could you remove this file ? (with ~)

**Comment by legalsylvain** [comment]
:+1: except minor remark. thanks for this module. (and test file). 

Just a question, how does analytic works by default for PoS. (I don't use analytic). Which account is used ? products setting / other / nothing ?

regards.

**Comment by pedrobaeza** [comment]
The name can be only _pos_analytic_, because there's no other module that handles analytic in POS, and we are putting all the transversal modules that handles analytic stuff to OCA/account-analytic, like the one for MRP that I'm doing or the one for product from Alexis.

**Comment by adrienpeiffer** [comment]
Thanks !
@legalsylvain without development, there is no analytic account.
https://github.com/odoo/odoo/blob/8.0/addons/point_of_sale/point_of_sale.py#L1084

File: pos_analytic_config/README.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+------------
+
+* Adrien Peiffer <adrien.peiffer@acsone.eu>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```
Comment (elicoidal) [inline pos_analytic_config/README.rst]: http (x2)

File: pos_analytic_config/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline pos_analytic_config/__openerp__.py:1]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: pos_analytic_config/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+##############################################################################
+{
+    'name': "POS Analytic Config",
+
+    'summary': """Use analytic account defined on
+                  POS configuration for POS orders""",
+    'author': 'ACSONE SA/NV,'
+              'Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
+    'category': 'Point Of Sale',
```
Comment (elicoidal) [inline pos_analytic_config/__openerp__.py]: add 'accounting'?

**Comment by legalsylvain** [comment]
@pedrobaeza : Not sure to be agree with your PoV. 
Another possible implementation could be to reuse product analytic config. So a pos_analytic_product is possible. (based on account_analytic_default) Don't you think ?
Set all pos sales in one single analytic account is a choice.

Regards.

**Comment by pedrobaeza** [comment]
Yeah, and that module can also be hosted on account-analytic.

**Comment by legalsylvain** [comment]
And yes, if MRP and product are done in another repository, this PR should be done against

**Comment by Cedric-Pigeon** [comment]
:+1: (Code review)

File: pos_analytic_config/__openerp__.py (hunk 5 above)
Comment (adrienpeiffer) [inline pos_analytic_config/__openerp__.py:1]: Done. Thanks !

**Comment by adrienpeiffer** [comment]
The discussion will continue on account-analytic.Thanks to all !

**Pull request closed without merging**